package.json

```diff
@@ -112,4 +112,4 @@
     "proxyquire": "1.8.0",
     "rimraf": "2.6.1"
   }
-}
+}
```

Comment:
I noticed your editor removed the trailing newline character from both these files.  Those newlines need to be preserved for TSLint and so would cause problems if you had edited a .ts file.  The project's .editorconfig file specifies that a newline should be added to a file on save, which makes me think your editor is not configured to recognize the .editorconfig file.  If you're using VSCode, you can install the EditorConfig plugin to fix this.